src/screens/settingscreen/SettingScreen.tsx

```diff
@@ -4,6 +4,7 @@ import { StackNavigationProp } from "@react-navigation/stack";
 import {
   OptionsList,
   OptionItem,
+<<<<<<< HEAD
```

Comment:
I think you forgot to complete the merge here? There's still merge comments from git.